config/test.json

```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "./default.schema.json",
 	"host": "localhost",
-	"port": 3031,
```

Comment:
we use port 3031 because of rancher desktop having the port 3030 bocked. So we use port 3031 for local development/testing but push port 3030 for everyone else. the fact that you see some change here is just a little mishap ;)